web-design/02_typography.md

```diff
@@ -0,0 +1,85 @@
+<div align="center">
+    <img src="../images/typography.png" alt="Logo" height="170" align="center">
+    <h1 align="center">Typography</h1>
+</div>
+
+#### 🕒 Estimated time: 8 h
+## 📝 Description
+> Typography is the art and technique of arranging type to make written language legible, readable, and appealing when displayed. The arrangement of type involves selecting typefaces, point sizes, line lengths, line-spacing (leading), and letter-spacing (tracking), and adjusting the space between pairs of letters (kerning). Typography also includes the use of color, images, and other visual elements to enhance the appearance and readability of text.
+
+
+## 📋 Prerequisites
+- [ ] [Introduction to web design](./01_web-design-concepts.md).
+
+## 🎯 Learning Objectives
+- [ ] Understand the principles of typography and its role in visual communication.
+- [ ] Learn how to select appropriate typefaces for different design contexts.
+- [ ] Explore techniques for effective typography hierarchy and readability.
+- [ ] Gain knowledge of typographic terminology and best practices for spacing and alignment.
```

Comment:
remove these `[ ]` and also some of these `-` and try to keep your readmes simple as much as possible.
For reference, you can have a look at the latest update of my readme files.
- Do this with all of your files.

Reply:
Hello @ahzamir 
Thanks for your kind review, I will apply the requested changes